src/jit/emit.h

```diff
@@ -1327,10 +1327,12 @@ class emitter
     insUpdateModes emitInsUpdateMode(instruction ins);
     insFormat emitInsModeFormat(instruction ins, insFormat base);
 
+#if !defined(_TARGET_XARCH_)
```

Comment:
This appears to be xarch-only, so should just be removed.

Reply:
Fixed.